model.py

```diff
@@ -5989,9 +5989,12 @@ def get(cls, _db, url, do_get=None, extra_request_headers=None,
         :return: A 2-tuple (representation, obtained_from_cache)
 
         """
-        do_get = do_get or cls.simple_http_get
-
         representation = None
+        request_kwargs = {}
+        if params:
+            do_get = do_get or cls.simple_http_post
```

Comment:
This works, but using a POST to simulate GET because we know the result is cacheable is weird enough that I'd prefer to do everything in the do_post() method.